### ISSUE TITLE
Portable GUI Closes Itself After Certain Distance

### ISSUE THREAD

**Issue by Jonathanace** [benbenlaw/opolisutilities#7]
Portable GUI works when near a chest, but after a certain distance, the GUI closes itself immediately upon opening. 

Video: https://medal.tv/games/minecraft/clips/j7rlcMhIxBEsWAgKr?invite=cr-MSxpUHIsMjY4NTkyNzAzLA

**Comment by benbenlaw** [comment]
yeah theres an issue when using it out side of single player chunk renering range at the moment

**Comment by Jonathanace** [comment]
@benbenlaw I did some more testing and found that the portable GUI does not have this issue with vanilla chests and furnaces, only the "Iron Chests" mod upgraded (iron, gold, diamond, etc.) chests.

**Comment by benbenlaw** [comment]
yeah pretty sure vanilla stuff works fine its just modded

**Comment by benbenlaw** [comment]
cant even get it to work with my own mods lol
